syntax_and_semantics/annotations.md

```diff
@@ -1,6 +1,6 @@
-# Attributes
+# Annotations
 
-Some types and methods can be annotated with attributes. The attribute list is fixed, but eventually (maybe) there will be user-defined attributes.
```

Comment:
This should also mention that some user-defined annotations are already provided by the stdlib. Eventually there should also be a list of those. For a developer using Crystal there is little practical difference whether a annotation is provided by the compiler or stdlib. But this can come later.